general/hash.hpp

```diff
@@ -57,9 +57,9 @@ struct Hashed4
  *  each time this class is invoked.
  *
  *  There are two main methods this class provides. The Get(...) methods always
- *  return an item given the two or four indices. If the item didn't previously
+ *  return an item given the two or four indices. If the item did not previously
```

Comment:
Out of curiosity, what prompted the changes to this file (which is seemingly independent of the rest of the PR)?

Reply:
This is used in several places throughout this file from many years ago, and I just copied it here. `ParNCMesh::ElementNeighborProcessors` appends to the array `ranks`, and this reserves memory for 64 ranks. I don't know how this was originally chosen, but I suppose it is an optimization to avoid reallocating this array until it exceeds size 64. It may not matter much, since this is unlikely to be a bottleneck, but I suppose some choice for the initial size should be made, and 64 is reasonable. It must be at least 2 in parallel, and 64 is not very large. So the answer is, why not 64? Unless someone has a better explanation.